.github/workflows/ci.yaml

```diff
@@ -7,10 +7,14 @@
 name: CI
 
 # This workflow will run when developer push a topic branch to their
-# fork in github, minimizing noise for maintainers.
+# fork in github, minimizing noise for maintainers;
+# It also runs on nightly basis at 12:00 AM
 on:  # yamllint disable-line rule:truthy
   - push
   - pull_request
+  - workflow_dispatch:
+    schedule:
+      - cron:  '*/5 * * * *'
```

Comment:
I still see an error in your fork - the yaml is invalid:

> Annotations
> 1 error
> [Error: .github#L1](https://github.com/Shwetha-Acharya/ramen/commit/95dad519fff30172274539437acab23bb865a2b0#annotation_12560015663)
Invalid type for `on`

The example from vdsm project is:

```
on:
  # Be able to run the job manually when needed
  # (Actions -> Containers -> Run workflow)
  workflow_dispatch:
  # Build every week on Sunday 00:00
  schedule:
    - cron:  '0 0 * * 0'
```

I think this should work:

```
on:
  push:
  pull_request:
  schedule:
    - cron:  '0 0 * * 0'
```

See the docs linked from #566